docs/installation/manage/_index.md

```diff
@@ -109,50 +146,61 @@ brew services restart viam-server
 {{% /tab %}}
 {{< /tabs >}}
 
-## Troubleshooting
+## View `viam-server` logs
+
+`viam-server` writes log messages as it starts up and runs, providing useful information when managing or troubleshooting the process.
+Use the following commands to view these log messages locally on your system.
 
-{{< tabs name="Troubleshooting">}}
-{{% tab name="Linux" %}}
+{{< alert title="Tip" color="tip" >}}
+If your system is able to connect with the Viam app, logs can also be viewed in the **Logs** tab on [the Viam app](https://app.viam.com/).

```

Comment:
```suggestion
If your system is able to connect with the Viam app, you can also view logs in the **Logs** tab on [the Viam app](https://app.viam.com/).
```

Have you installed vale in vscode? It should be underlining all of these passive voices

Reply:
Installed but ... was disabled! Enabled now, thank you!